recipes/conda-rpms/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "conda-rpms" %}
+{% set version = "1.0" %}
+{% set sha256= "7f3b02a5aacd3fe69cd2703862cddc70f8fdd6e11b9c4ac4420b4af43babd7b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/SciTools-incubator/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - jinja2
+    - gitpython
+    - conda-build-all
+    - pyyaml
```

Comment:
I have not added `mock` (see [conda-rpms requirements.txt](https://github.com/SciTools-incubator/conda-rpms/blob/master/requirements.txt)) as it is only a testing dependency